README.md

```diff
@@ -24,3 +40,28 @@ ng-models -m misc --host 127.0.0.1 -d automated_test -u postgres -p postgres
 ```
 
 The examples above show how you can either: 1) supply the environment variables or 2) manually input them via parser arguments.
+
+    
+## Setup pre-commit hook to check for secrets
+
+We use [pre-commit](https://pre-commit.com/) to setup pre-commit hooks for this repo.
+We use [detect-secrets](https://github.com/Yelp/detect-secrets) to search for secrets being committed into the repo. 
+
+To install the pre-commit hook, run
+```
+pre-commit install
+```
+
+To update the .secrets.baseline file run
+```
+detect-secrets scan --update .secrets.baseline
+git add .secrets.baseline
```

Comment:
don't need the `git add` line as this commit is already adding the baseline file.